test/IISExpress.FunctionalTests/InProcess/TestServerTest.cs

```diff
@@ -1,32 +1,37 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
```

Comment:
Can we separate the tests into two different classes? One for TestServer, the next for client disconnect?